README.md

```diff
@@ -44,6 +44,21 @@ following actions on the project and its submodules:
 
 > **_NOTE:_** The minimum supported Gradle version to use this plugin is v6.9
 
+<details>
+<summary> 🚚 Migrating from v4 </summary>
+
+### Breaking Changes 🚨
+
+* The minimum version of Gradle required to use this plugin has been upgraded to version 6.9.
+* Artifactory's convention attributes have been removed:
+
+  | Attribute |                                                                                     Migration action                                                                                      |
+  |:---------:|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+  |  parent   |                                                                        No longer supporting legacy configurations.                                                                        |
+  |  resolve  | Refer to the [documentation](https://docs.gradle.org/current/userguide/declaring_repositories.html#declaring-repositories) for different ways to configure your repositories with Gradle. |
+
```

Comment:
Let's remove this table, and add the following billets instead.

* The *parent* attribute is no longer supported.
* The *resolve* attribute is no longer supported. To define the Artifactory resolution reposotpries for your build, declate the repositories under the *repositories* section as described [here](https://docs.gradle.org/current/userguide/declaring_repositories.html#declaring-repositories)
.